### PR TITLE
Fix the regex to extract the pageId from the page source

### PR DESCRIPTION
**Pull request by nanux**
Confluence 8.1 introduced additional whitespace into the HTML tags; we need to change the regular expression extracting the `pageId` to account for it.

**Version 8.0**
```
<a  id="editPageLink" href="/wiki/pages/editpage.action?pageId=1004923276"
```

**Version 8.1**
```
<a  id="editPageLink"                href="/wiki/pages/editpage.action?pageId=1004923276"
```

### CODE REVIEW

File: app/jmeter/confluence.jmx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.4.3">
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.5">
```
Comment (nanux) [inline app/jmeter/confluence.jmx:2]: I have the 5.5 `jmeter`, I am not sure if this will be a problem. Please let me know if I should revert the change.

**Review by jjeongatl — APPROVED** [review]

File: app/jmeter/confluence.jmx (hunk 1 above)
Comment (ometelytsia) [reply app/jmeter/confluence.jmx:2]: Should not be a problem, but I would suggest keeping 5.4.3 here for consistency, as `confluence.yml` file has this version pinned.

**Review by ometelytsia — APPROVED** [review]

**Review by SergeyMoroz0703 — APPROVED** [review]
